src/components/department_card.tsx

```diff
@@ -0,0 +1,101 @@
+import Link from "next/link";
```

Comment:
In my opinion, a better casing for this file would be kebab-case. Resulting in `department-card`. This is a standard that is starting to get some adoption 👀

Reply:
Agreed.